cmd/fetch.go

```diff
@@ -109,8 +139,8 @@ func (w *writerWorkerStruct) card() {
 			continue
 		}
 		var buffer bytes.Buffer
-		cardName := fmt.Sprintf("%v-%v%v-%v.json", card.Set, card.Side, card.Release, card.ID)
-		dirName := filepath.Join(card.Set, fmt.Sprintf("%v%v", card.Side, card.Release))
+		cardName := fmt.Sprintf("%v-%v-%v.json", card.Set, card.Release, card.ID)
```

Comment:
Why removing the `.Side` ? It's important for the code.

Reply:
Because the `.Release` now includes the `Side` information, if it's in the normal card code. Some cards don't have the side in their card code. For example, the side for https://en.ws-tcg.com/cardlist/list/?cardno=BSF2024-03%20PR is W, but that's not in the card code ("BSF2024-03 PR"). In order to make the file name consistent with the card code, I made changes in kib.go to put the full release string in `.Release` and not exclude the first character from `.Release`. Since `.Release` includes the side in normal cases, there's no need to include it here again. If we do, for special cases, such as the example, we'd end up writing a string that isn't consistent with the card code (ie. "WBSF2024-03 PR" for the given example).

Reply:
This data is used in other tools so I don't like we change all the cards for one exception.
English side is notorious to do weird thing, I think it's better to handle it explicitly in the code instead of this.